### PR TITLE
Ch integrate heroku

### PR DESCRIPTION
**Pull request by Mbarak-Mbigo**
**What does this PR do?**
It integrates heroku deployment

**Description of Task to be completed?**
Integrate local repo to heroku (deployment)

**How should this be manually tested?**
visit `https://wgervulcan.herokuapp.com``

**Any background context you want to provide?**
None

**What are the relevant pivotal tracker stories?**
#142465231

### CODE REVIEW

File: wger/settings.py [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+ADMINS = (
+    ('Your name', 'your_email@example.com'),
+)
+MANAGERS = ADMINS
+
+
+DATABASES = {
+    'default': {
+        'ENGINE': 'django.db.backends.sqlite3',
+        'NAME': '/Users/mbarakmbigo/wger_vulcan/database.sqlite',
```
Comment (amwaleh) [inline wger/settings.py]: This has your development/personal settings in it. Let the name of the database be retrieved from  .env

File: wger/settings.py [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+# Your reCaptcha keys
+RECAPTCHA_PUBLIC_KEY = ''
+RECAPTCHA_PRIVATE_KEY = ''
+NOCAPTCHA = True
+
+# The site's URL (e.g. http://www.my-local-gym.com or http://localhost:8000)
+# This is needed for uploaded files and images (exercise images, etc.) to be
+# properly served.
+SITE_URL = 'http://localhost:8000'
```
Comment (amwaleh) [inline wger/settings.py]: This might also need to change once on heroku. But am not sure !! please prove me wrong.

File: wger/settings.py [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+
+# Path to uploaded files
+# Absolute filesystem path to the directory that will hold user-uploaded files.
+MEDIA_ROOT = '/Users/mbarakmbigo/.local/share/wger/media'
```
Comment (amwaleh) [inline wger/settings.py]: Yet again your dev setting have been hard coded

**Review by amwaleh — CHANGES_REQUESTED** [review]
Great work!! just make sure your development settings are not hard coded, try using a `.env` file.

File: wger/settings.py (hunk 3 above)
Comment (Mbarak-Mbigo) [reply wger/settings.py]: These settings were generated during installation as I followed the app's readme from the github repo. I will change accordingly.

Comment (amwaleh) [reply wger/settings.py]: Then I believe this file need to be git ignored

**Review by amwaleh — APPROVED** [review]

**Pull request merged**